docs/features/kubernetes/configuration.md

```diff
@@ -402,6 +402,41 @@ view the Kubernetes API docs for your Kubernetes version (e.g.
 [API Groups for v1.22](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#-strong-api-groups-strong-)
 )
 
+### `objectTypes` (optional)
+
+Overrides for the Kubernetes object types fetched from the cluster. The default object types are:
+
+- pods
+- services
+- configmaps
+- limitranges
```

Comment:
You need to add these terms to `.github/vale/Vocab/Backstage/accept.txt` for the pipeline to pass 😄